.pre-commit-config.yaml

```diff
@@ -18,6 +18,7 @@ repos:
         args:
           - --allow-missing-credentials
       - id: detect-private-key
+        exclude: src/secrets/privkey.pem
```

Comment:
Can you please either create an issue or start a draft skeleton-docker PR to remove this?  That will give us something concrete so that we won't lose this after I resolve this conversation.

Reply:
Please see https://github.com/cisagov/skeleton-docker/pull/115 which includes this change.

Reply:
Awesome, thank you!